StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -553,13 +553,13 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
     }
 
     func consumerSessionStartVerification(
-        otpType: String, // TODO(kgaidis): consider whether this should be an enum type SMS + EMAIL
```

Comment:
these TODO's were noise in comparison to important TODO's so I deleted them...the road to Link API's have been bit more tricky than originally expected so this resolve over-time naturally when we figure out a 'final' solution